soccer/main.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
Maybe instead of using `__future__` we can stick with `click.echo`. It seems to support Python 2 and 3, so it should take care of this. [docs](http://click.pocoo.org/5/quickstart/#echoing)